server/src/portfolioRouter.mjs

```diff
@@ -9,10 +9,39 @@ router.get("/", async (request, response) => {
   response.json(portfolios);
 });
 
+router.get("/:portfolioID", async (request, response) => {
```

Comment:
These methods should probably have some error handling for if the requested object is not found or the write fails